src/components/charts/lightCurve/light-curve.module.scss

```diff
@@ -9,6 +9,13 @@
   }
 }
 
+.padded-drawer-inner {
+  padding-right: $minPadding;
+  padding-bottom: $minPadding;
+  padding-left: 72px;
+  outline: none;
+}
+
```

Comment:
Unless there's some conflict, I'd like to use the `padded-drawer-inner` class in `charts/shared/navDrawer/nav-drawer.module.scss` instead of duplicating the styles here.  Since that's like a "shared" style we're applying in a lot of the implementations of this NavDrawer.